runtime/statemine/src/lib.rs

```diff
@@ -42,36 +42,40 @@ use sp_version::RuntimeVersion;
 use frame_system::{
 	limits::{BlockLength, BlockWeights},
 };
-use runtime_common::{
-	BlockNumber, Signature, AccountId, Balance, Index, Hash, NORMAL_DISPATCH_RATIO,
+pub use runtime_common::{
```

Comment:
```suggestion
use runtime_common::{
```

not that if you want to use any of this in chain_spec, I exported all as `common`, so this doesn't need to be `pub` anymore.

Reply:
Good point, I'll do this change.